File: td-bigdata-openapi-apptest/src/main/scala/akkaApp/HelloMainSimple.scala
package akkaApp

import akka.actor.{ActorRef, Props, ActorSystem}


/**
  * Created by Administrator on 2016/3/25.
  */
object HelloMainSimpleForScala {

  def main(args: Array[String]) {
    val _system = ActorSystem("HelloForScala")
    val helloWorldActorRef:ActorRef = _system.actorOf(Props[HelloWorldActor])
    println("HelloWorld Actor path:"+helloWorldActorRef.path)

  }

}

File: td-bigdata-openapi-apptest/src/main/java/akkaApp/HelloMainSimple.java
package akkaApp;

import akka.actor.ActorRef;
import akka.actor.ActorSystem;
import akka.actor.Props;
import com.typesafe.config.ConfigFactory;

/**
 * Created by Administrator on 2016/3/25.
 */
public class HelloMainSimple {

    public static void main(String[] args){
        ActorSystem system = ActorSystem.create("Hello", ConfigFactory.load("samplehello.conf"));
        ActorRef a = system.actorOf(Props.create(HelloWorld.class),"helloWorld");
        System.out.println("HelloWorld Actor Path:" + a.path());
    }
}
